extension.js

```diff
@@ -296,7 +319,7 @@ const BingWallpaperIndicator = new Lang.Class({
             }
         } else {
             this.title = _("No wallpaper available");
-            this.explanation = _("No picture for today 😞.");
+            this.explanation = _("No picture for today ðŸ˜ž.");
```

Comment:
Is this emoticon value expected? 😃

Reply:
Sorry for that, my editor may do not support that encode .......